src/style.css

```diff
@@ -1,3 +1,61 @@
+html {
+  font-size: 12px;
+  font-weight: 500;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding:0 50px;
+}
+
+.logo_link {
+  width: 40px;
+  height: 40px;
+  }
+
+.list__items {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+}
+
+.list__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  position: relative;
+  }
+
+.nav__link:hover,
+.is-active {
+  color: #00ACDC;
```

Comment:
- CSS Variable is used for a blue color
![image](https://github.com/mate-academy/layout_moyo-header/assets/63243136/6ac30e8d-dff1-4c14-a1c8-be56419db70a)